src/music/midifile.cpp

```diff
@@ -21,6 +21,13 @@
 #include "../console_func.h"
 #include "../console_internal.h"
 
+template <typename T>
+T* grow(std::vector<T>& vec, std::size_t size)
```

Comment:
"size" is a bit generic here, especially with size_t & size() usages, perhaps "amount" ?
also, doxygen?